maplibre-demo/src/main.rs

```diff
@@ -117,5 +102,5 @@ fn main() {
     enable_tracing();
 
     run_headless();
-    run_in_window();
+    //run_in_window();
```

Comment:
```suggestion
    run_in_window();
```

I think we can leave this one uncommented on the main branch.